start.sh

```diff
@@ -1,115 +1,104 @@
 #!/usr/bin/env bash
 set -ex
 
-GROUP=staff
-USER_UID=1001
-export HOME="/home/$USER"
+USER="rstudio"
+USER_UID="1000"
+USER_HOME=$(getent passwd "${USER_UID:?}" | cut -d: -f6)
 
 function cleanup() {
- rm -rf "$HOME/.conda/envs/rstudio"
+  if [[ -d "${USER_HOME}/.conda/envs/rstudio" ]];then
+    rm -rf "${USER_HOME}/.conda/envs/rstudio"
+  fi
 }
+
 trap cleanup ERR
+source functions.sh
 
 function init_user() {
-  if [ "$(id -u "$USER" 2>/dev/null)" != $USER_UID ]; then
-      useradd -g $GROUP -u $USER_UID -d "/home/$USER" "$USER" || true
-  fi
-
-  # this is only for running locally - in-cluster, home directories are
-  # provided via NAS
-  echo "${USER}:${USER}" | chpasswd
-  if [ ! -d "/home/$USER" ]; then
```

Comment:
Only if we move to using the included scripts. 

It looks like it might be taken straight fromn the user_init script actually

```bash
# File: /etc/cont-init.d/userconf

## Add a password to user
echo "$USER:$PASSWORD" | chpasswd
```